setup.py

```diff
@@ -151,7 +154,7 @@ def get_source_files():
     long_description = fp.read()
 
 setup(
-    name = 'pylibjpeg-libjpeg',
+    name = 'pymedphys-pylibjpeg-libjpeg',
```

Comment:
?

Reply:
This wasn't supposed to be a PR sorry. This was just as a stop gap to get PyMedPhys working.